frontend/package.json

```diff
@@ -33,6 +33,8 @@
     "test:coverage": "npx lerna run test:coverage --stream --concurrency 1",
     "bf-applicant:test": "yarn --cwd benefit/applicant test",
     "bf-handler:test": "yarn --cwd benefit/bf-handler test",
+    "bf-applicant:browser-test": "yarn --cwd benefit/applicant browser-test",
+    "bf-applicant:browser-test:ci": "yarn --cwd benefit/applicant browser-test:ci",
```

Comment:
you probably have to add some certificate flags to make the tests work on review environment. See the same script on kesaseteli-employer

Reply:
Copy-pasted from Kesäseteli 🙈